src/platforms/python/common/configuration/integrations/loguru.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Loguru
+description: Learn about using Sentry with loguru.
```

Comment:
```suggestion
description: Learn about using Sentry with Loguru.
```

Reply:
Fixed in 63f4e09e46f0ad7eb6fa2f46769f376780289f68. Thank you!